packages/shopping-cart/src/types.ts

```diff
@@ -569,7 +569,7 @@ export interface ResponseCartProductExtra {
 	 */
 	is_marketplace_product?: boolean;
 	product_slug?: string;
-	product_type?: 'marketplace_plugin' | 'marketplace_theme';
+	product_type?: 'marketplace_plugin' | 'marketplace_theme' | 'saas_plugin' | 'saas_theme';
```

Comment:
I appreciate trying to be future-proof but we don't have a `saas_theme` product type yet and haven't seen such a product generally. I would suggest removing `saas_theme`.